packages/core/src/renderer/components/+namespaces/namespace-store.test.ts

```diff
@@ -137,6 +153,10 @@ describe("NamespaceStore", () => {
     ]);
   });
 
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
This should not be necessary. This implies that there is shared state between tests.

Reply:
This is needed to reset

```
deleteNamespaceMock = jest.spyOn(NamespaceApi.prototype, "delete").mockImplementation();
```

Reply:
A better way to move around this would be to do the following:

```ts
beforeEach(() => {
  deleteNamespaceMock = jest.fn();
  di.override(namespaceApiInjectable, () => ({
    delete: deleteNamespaceMock,
  } as any);
});
```